.mvn/wrapper/maven-wrapper.properties

```diff
@@ -1,4 +1,4 @@
 # custmized maven that help avoid download only highest poms from version range
 # https://issues.apache.org/jira/browse/MRESOLVER-164
-distributionUrl=https://confluent-packaging-tools.s3-us-west-2.amazonaws.com/apache-maven-3.6.3.1-bin.zip
+distributionUrl=https://confluent-packaging-tools.s3-us-west-2.amazonaws.com/apache-maven-3.8.1.2-bin.zip
```

Comment:
This is the version @xli1996 kindly built after merging https://github.com/confluentinc/maven/pull/1.